fastlane/Fastfile

```diff
@@ -22,151 +22,157 @@ system("/usr/libexec/PlistBuddy -c \"add :ConnectHardwareKeyboard bool 0\" ~/Lib
 system("rm -r $HOME/Library/Developer/Xcode/DerivedData/SalesforceMobileSDK-*")
 
 def combinePackagesOfXML(toFile, files)
-	toDoc = Nokogiri::XML(File.open(toFile))
-	packageNode = toDoc.xpath('//packages/package')
-	packageNode = packageNode[packageNode.size-1] #get last package node
-	for fromfile in files
-		fromDoc = Nokogiri::XML(File.open(fromfile))
-		packages = fromDoc.xpath('//packages/package')
-		for package in packages
-			packageNode.add_next_sibling(package)
-		end
-	end
-	File.write(toFile, toDoc.to_xml)
+  toDoc = Nokogiri::XML(File.open(toFile))
+  packageNode = toDoc.xpath('//packages/package')
+  packageNode = packageNode[packageNode.size-1] #get last package node
+
+  for fromfile in files
+    fromDoc = Nokogiri::XML(File.open(fromfile))
+    packages = fromDoc.xpath('//packages/package')
+    for package in packages
+      packageNode.add_next_sibling(package)
+    end
+  end
+  File.write(toFile, toDoc.to_xml)
 end
 
 def doDanger
-	if $Jenkins
-		begin
-			 danger(
-				 danger_id: "CI-result",
-				 dangerfile: "DangerFile",
-				 github_api_token: ENV["DANGER_GITHUB_API_TOKEN"],
-				 verbose: true
-			 )
-		 rescue => ex
-			 UI.error(ex)
-		 end
-	end
+  if $Jenkins
+    begin
+      danger(
+          danger_id: "CI-result",
+          dangerfile: "DangerFile",
+          github_api_token: ENV["DANGER_GITHUB_API_TOKEN"]
+      )
+    rescue => ex
+      UI.error(ex)
+    end
+  end
 end
 
 
 def testScheme(scheme)
-	scan(
-		workspace: "SalesforceMobileSDK.xcworkspace",
-		scheme: scheme,
-		device: ENV["device"],
-		output_directory: "test_output”,
-		code_coverage: true,
-		fail_build: false,
-		clean: true
-	)
-
-	begin
-		xcodebuild(
-		 	xcargs: "CLANG_ANALYZER_OUTPUT=plist-html CLANG_ANALYZER_OUTPUT_DIR=./clangReport RUN_CLANG_STATIC_ANALYZER=YES",
-		 	workspace: "SalesforceMobileSDK.xcworkspace",
-		 	scheme: scheme,
-		 	fail_build: false
-		)
-	rescue => ex
-		UI.error(ex)
-	ensure
-		#move clangReports to one folder
-		system("mkdir -p ../clangReport/StaticAnalyzer")
-		system("mv ../libs/SalesforceAnalytics/clangReport/StaticAnalyzer/SalesforceAnalytics ../clangReport/StaticAnalyzer/")
-		system("mv ../libs/SalesforceHybridSDK/clangReport/StaticAnalyzer/SalesforceHybridSDK ../clangReport/StaticAnalyzer/")
-		system("mv ../libs/SalesforceReact/clangReport/StaticAnalyzer/SalesforceReact ../clangReport/StaticAnalyzer/")
-		system("mv ../libs/SalesforceSDKCore/clangReport/StaticAnalyzer/SalesforceSDKCore ../clangReport/StaticAnalyzer/")
-		system("mv ../libs/SmartStore/clangReport/StaticAnalyzer/SmartStore ../clangReport/StaticAnalyzer/")
-		system("mv ../libs/SmartSync/clangReport/StaticAnalyzer/SmartSync ../clangReport/StaticAnalyzer/")
-	end
+  begin
+    xcodebuild(
+        xcargs: "CLANG_ANALYZER_OUTPUT=plist-html CLANG_ANALYZER_OUTPUT_DIR=./clangReport RUN_CLANG_STATIC_ANALYZER=YES",
+        workspace: "SalesforceMobileSDK.xcworkspace",
+        scheme: scheme,
+        sdk: "iphonesimulator",
```

Comment:
Specifying simulator was the only change necessary to get rid of code signing errors.